subsys/suit/cache/src/suit_dfu_cache_rw.c

```diff
@@ -45,23 +44,21 @@ LOG_MODULE_REGISTER(suit_cache_rw, CONFIG_SUIT_LOG_LEVEL);
 /* Adding 9 bytes for max length tstr header*/
 #define MAX_URI_ENCODE_BUFFER_LENGTH (CONFIG_SUIT_MAX_URI_LENGTH + 9)
 
-extern struct dfu_cache dfu_cache;
-static bool prevent_second_init;
-
 struct dfu_cache_partition_ext { /* Extended structure describing single cache partition */
 	const struct device *fdev;
+	size_t offset;
```

Comment:
What's the purpose of the `offset` field?
I see that it is used only inside preinitialize/cache_0_resize and not updated on I/O operations, so it would suggest that it is used only during the initialization.

Reply:
Discussed offline.